Licensing update for Plumber (#4373)
umbraco/UmbracoDocs: Add-ons/Umbraco-Plumber/Licensing/index.md

2. Copy the `test.lic` file  into `/App_plugins/Plumber`.

   ![Lic Placement](images/lic.png)
   

> # Getting an Umbraco Plumber License

Reviewer: ```suggestion
## Getting an Umbraco Plumber License
```

> If you want to buy a Plumber license, please reach out to the sales team through **suits@umbraco.com**.

Reviewer: Can you move this section below the Note to avoid confusion? 🙂

:::note
The test license is restricted to sites running in a development environment, with a debugger attached (ie hitting F5 in Visual Studio, in Debug mode, will enable the test license).
:::

## Getting an Umbraco Plumber License

> If you want to buy a Plumber license, please reach out to the sales team through **suits@umbraco.com**.

Reviewer: ```suggestion
If you want to buy an Umbraco Plumber license, reach out to the sales team at **suits@umbraco.com**.
```
